app/src/utils/services/DataService.ts

```diff
@@ -38,7 +38,7 @@ class DataService {
                 {key: 'task', value}
             ]).flat());
 
-        query.from(from);
+        // query.from(from);
```

Comment:
This will result in https://github.com/hochschule-darmstadt/einander-helfen/issues/88
If your local compilation fails please update your dependencies via `npm i` if this isn't working due to npm caching remove your node_modules dir and repopulate it via `npm i`